src/drivers/mct/main/seq_diag_mct.F90

```diff
@@ -1402,6 +1404,8 @@ subroutine seq_diag_ocn_mct( ocn, xao_o, frac_o, infodata, do_o2x, do_x2o, do_xa
        if (first_time) then
           index_x2o_Fioi_melth  = mct_aVect_indexRA(x2o_o,'Fioi_melth')
           index_x2o_Fioi_meltw  = mct_aVect_indexRA(x2o_o,'Fioi_meltw')
+          index_x2o_Fioi_bergh  = mct_aVect_indexRA(x2o_o,'PFioi_bergh')
```

Comment:
needs review.

Reply:
@billsacks can you have a look at these

Reply:
I am not very familiar with the seq_diag code. @alperaltuntas are you familiar with this? (I'm asking you because it seems like this code is mainly referenced by ocean folks... if you're not familiar with it either, do you have suggestions for who might be able to review it?)

That said: My initial concern is what happens in this code if (as will be the case for CESM) these coupling indices and/or the data stored within them are undefined.

Reply:
From @mvertens on telecon: Below, we should check if indices of these new variables are 0; if so, use the old calculation, if non-zero, use the new calculation.

@alperaltuntas no need for you to review further unless you want to.

Reply:
@mvertens , @billsacks , can you confirm my change below?

Reply:
It looks mostly right, but elsewhere where we have code like this, we use `perrWith='quiet'` in the call to mct_aVect_indexRA. See prep_rof_mod for an example. I think @mvertens suggested this to me, so I'll defer to her as to whether that's needed... I think it may be needed to avoid having mct die when you try to get a non-existent index?

Also, minor quibble: can you change `.eq.` to `==`? I don't think we're totally consistent within cime, but we tend to prefer `==` over `.eq.`, and at least try to stay consistent within a module.

Reply:
@billsacks , done